[["file:///home/jfvilladiego/proyectos_laravel/cotizaciones_vcb_laravel/vendor/symfony/process/Exception/LogicException.php",{"_uri":"file:///home/jfvilladiego/proyectos_laravel/cotizaciones_vcb_laravel/vendor/symfony/process/Exception/LogicException.php","_root":{"kind":4096,"name":"file:///home/jfvilladiego/proyectos_laravel/cotizaciones_vcb_laravel/vendor/symfony/process/Exception/LogicException.php","location":{"uriHash":1146699917,"range":{"start":{"line":0,"character":0},"end":{"line":21,"character":0}}},"children":[{"kind":512,"name":"Symfony\\Component\\Process\\Exception","location":{"uriHash":1146699917,"range":{"start":{"line":11,"character":0},"end":{"line":11,"character":46}}}},{"kind":1,"name":"Symfony\\Component\\Process\\Exception\\LogicException","location":{"uriHash":1146699917,"range":{"start":{"line":18,"character":0},"end":{"line":20,"character":1}}},"children":[],"associated":[{"kind":1,"name":"LogicException"},{"kind":2,"name":"Symfony\\Component\\Process\\Exception\\ExceptionInterface"}],"doc":{"description":"LogicException for the Process Component.","type":""},"modifiers":0}]},"_hash":1146699917}]]